src/plugins/discover/public/application/main/utils/get_fetch_observable.ts

```diff
@@ -53,7 +53,6 @@ export function getFetch$({
         );
       })
     ),
-    data.query.queryString.getUpdates$(),
```

Comment:
query change / update is now handled here:

https://github.com/elastic/kibana/pull/159903/files#r1236803970